src/main/java/frc/robot/subsystems/Arm/ArmSubsystem.java

```diff
@@ -55,16 +55,25 @@ public void periodic() {
 
     pidOutput = MathUtil.clamp(pidOutput, -1, 1);
     armIO.setArmSpeed(pidOutput);
+
     NetworkTableInstance.getDefault().getEntry("/Arm/ArmAngle").setNumber(armIO.getArmEncoderAngle());
     NetworkTableInstance.getDefault().getEntry("/Arm/pidOutput").setNumber(pidOutput);
     NetworkTableInstance.getDefault().getEntry("/Arm/armGoal")
         .setNumber(Units.radiansToDegrees(pidController.getGoal().position));
     armIO.update();
+
+    if (armIO.isEncoderConnected()) {
+      armIO.setArmSpeed(pidOutput);
+    }
+    else {
+      armIO.setArmSpeed(0);
+    }
```

Comment:
You can't shut down a system like this without creating an error dialog with the operator!

If you're at a competition there needs to be feedback to immediately trace why the subsystem is non operational.

At a minimum there should be a log message (ideally WARN: or ERROR: level) so that a programmer can swiftly diagnose the problem. IIRC this will be displayed in the driver station, correct?